src/aria-snapshots/menu-test-aria-ts-loading.yml

```diff
@@ -0,0 +1,2 @@
+- button "Toggle"
+- menu "Toggle"
```

Comment:
It's not clear to me why "Loading" doesn't show up in the tree. It shows up in DevTools and it's picked up by VoiceOver.

Reply:
Component not updated yet?

Reply:
Nah, the build would have failed if it weren't. I double checked just to be sure.